src/presentation/dtos/health-dto.ts

```diff
@@ -0,0 +1,11 @@
+export class HealthDto {
```

Comment:
This isn't used anywhere, so it should be deleted :)

Reply:
Deleted in the SOF-1967 branch